integration_test/cases/pool.exs

```diff
@@ -16,9 +16,7 @@ defmodule Ecto.Integration.PoolTest do
 
   test "can start multiple repos" do
     # Can't start a second Repo with the default name
-    {_, pool_name, _} = PoolRepo.__pool__
-    pool_pid = Process.whereis(pool_name)
-    assert {:error, {:already_started, ^pool_pid}} = PoolRepo.start_link()
+    assert {:error, {:already_started, _}} = PoolRepo.start_link()
```

Comment:
As long as the pool name is the one coming from the `__pool__` tuple, we are good to go.